README.md

```diff
@@ -0,0 +1,108 @@
+########################################################
+#			Welcome to the Kokkos Documentation Workflow     #
+########################################################
+
+	Here, we summarize the steps for setting up the kokkos-core-wiki repo, and a
+branch-based workflow for updating documentation in the main Kokkos project.  The kokkos-core-wiki is the
+canonical source of Kokkos documentation "Truth".
+
+	CAVEAT: Creation of wholly new pages will require Review and Approval in some
+cases.  Check with your team lead, if you are uncertain.
+
+	STEP 1) Create your own fork of the kokkos-core-wiki repo by navigating to
+					https://github.com/kokkos/kokkos-core-wiki, and clicking "Fork" in the upper
+					right of the github page.
+
+	STEP 2) Clone your forked repo:
+
+					git clone git@github.com:ajpowelsnl/kokkos-core-wiki.git
+
+	STEP 3) Create a topic branch in your local fork of the kokkos-core-wiki:
+
+					git checkout -b demo/setup
```

Comment:
I would differentiate between one-time setup steps and the ones that you need to do for every pull request. This one should be before the current "STEP 10)" IMHO.